docs/site/migration/auth/example.md

```diff
@@ -154,24 +154,46 @@ strategy to verify a user's identity. The authentication setup is borrowed from
 The authentication system aims to understand **who sends the request**. It
 retrieves the token from a request, decodes the user's information in it as
 `principal`, then passes the `principal` to the authorization system which will
-decide the `principal`'s access later. You can enable the jwt authentication by:
+decide the `principal`'s access later.
+
+To simplify the implementation for readers, we extracted the services and
+bindings into a component in folder `src/component`, it consists of the
+following files:
 
 - creating the jwt authentication strategy to decode the user profile from
   token. See
-  [file src/services/jwt.auth.strategy.ts](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration/src/services/jwt.auth.strategy.ts)
+  [file src/component/jwt.auth.strategy.ts](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration/src/component/jwt.auth.strategy.ts).
```

Comment:
To be consistent with other folder naming convention, wouldn't it be `components` instead of `component`?

Reply:
good point, refactored the casbin-auth and jwt-auth services into 2 components:

    /components
        /casbin-authorization
            /services
                /...files
        /jwt-authentication
            /services
                /...files